Gems/Atom/RHI/Vulkan/Code/Source/RHI/Instance.cpp

```diff
@@ -126,11 +127,33 @@ namespace AZ
             m_instanceCreateInfo.ppEnabledExtensionNames = m_descriptor.m_requiredExtensions.data();
 
             //For instance creation/destruction use nullptr for VkAllocationCallbacks* as using VkSystemAllocator::Get() crashes RenderDoc when used with openxr
-            //enabled projects. We think its because RenderDoc maybe injecting something when doing allocations. Using nullptr when USE_RENDERDOC or enableRenderDoc
-            //is enabled is another option but it will not work for Android easily and will require further work, not to mention manually enabling this for Android renderdoc. 
-            if (m_context.CreateInstance(&m_instanceCreateInfo, nullptr, &m_instance) != VK_SUCCESS)
+            //enabled projects. We think its because RenderDoc is maybe injecting something when doing allocations. Using nullptr when USE_RENDERDOC or enableRenderDoc
+            //is enabled is another option but it will not work for Android easily and will require further work, not to mention manually enabling this for Android renderdoc.
+            VkResult result = m_context.CreateInstance(&m_instanceCreateInfo, nullptr, &m_instance);
+
+            if (validation != RHI::ValidationMode::Disabled &&
+                (result == VK_ERROR_LAYER_NOT_PRESENT || result == VK_ERROR_EXTENSION_NOT_PRESENT))
+            {
+                // Remove all validation layers and extensions and try again to create the Vulkan instance.
+                // Some drivers report the validation layers as available but they fail to load them when creating the instance.
+                AZ_Warning("Vulkan", false, "Disabling validation due to Instance creation failure. Error = '%s'.", GetResultString(result));
+                RawStringList validationLayers = Debug::GetValidationLayers();
+                RemoveRawStringList(m_descriptor.m_requiredLayers, validationLayers);
+                RawStringList validationExtension = Debug::GetValidationExtensions();
+                RemoveRawStringList(m_descriptor.m_requiredExtensions, validationExtension);
+
+                m_descriptor.m_validationMode = RHI::ValidationMode::Disabled;
+                m_instanceCreateInfo.enabledLayerCount = static_cast<uint32_t>(m_descriptor.m_requiredLayers.size());
+                m_instanceCreateInfo.ppEnabledLayerNames = m_descriptor.m_requiredLayers.data();
+                m_instanceCreateInfo.enabledExtensionCount = static_cast<uint32_t>(m_descriptor.m_requiredExtensions.size());
+                m_instanceCreateInfo.ppEnabledExtensionNames = m_descriptor.m_requiredExtensions.data();
+
+                result = m_context.CreateInstance(&m_instanceCreateInfo, nullptr, &m_instance);
+            }
+
+            if (result != VK_SUCCESS)
             {
-                AZ_Warning("Vulkan", false, "Failed to create Vulkan instance");
+                AZ_Warning("Vulkan", false, "Failed to create Vulkan instance. Error = '%s'", GetResultString(result));
```

Comment:
You can use a raw string literal to have double quotes embedded in the string without using backslashes.
That way the result string is quoted using double quotes.
```suggestion
                AZ_Warning("Vulkan", false, R"(Failed to create Vulkan instance. Error = "%s")", GetResultString(result));
```